docs/data/charts/legend/legend.md

```diff
@@ -59,3 +59,52 @@ The `labelStyle` property is needed to measure text size, and then place legend
 Style applied by other means will not be taken into account.
 Which can lead to label overflow.
 :::
+
+## Color legend
+
+To display legend associated to a [colorMap](https://mui.com/x/react-charts/styling/#values-color), use the `<ContinuousColorLegend />` `'continuous'` color mapping or the `<PiecewiseColorLegend />` for color mapping of type `'piecewise'`.
+
+### Select data
+
+To select the color mapping to represent, use the following props:
+
+- `axisDirection` can be `'x'`, `'y'`, or `'z'`. It indicates which axis contain the `colorMap` definition.
+- `axisId` The id of the axis to use in case the selected direction contains multiple ones.
+
+{{"demo": "BasicColorLegend.js"}}
+
+### Position
+
+This component position is done exactly the same way as the [legend for series](#position).
```

Comment:
I did not found a good way to place it. For now the page is structure with the series legend on top as if it was the only one. Which is true for most of the users.

Then there is the section about color legends.

To put the positioning at level h2, it would require to quickly introduce the idea that we have multiple legends. Which IMHO introduce too much info for most of the users

> I believe this section is useless since the related doc is in this same page.

I would not say useless, but redundant. It does not bring new information. But without the section it's not obvious that you can move this legend

Reply:
Yep, `useless` was a poor choice of words. 😓